### ISSUE TITLE
Payment Methods: Unable to add PayPal despite PayPal being available.

### ISSUE THREAD

**Issue by arinoch** [Automattic/wp-calypso#56743]
### Quick summary

When navigating to Purchases > Payment Methods, you only have the option to add a credit card. Even in countries where other payment methods are available, the option for credit card is the only one.

### Steps to reproduce

1: Either create, or log into, a free account.
2: Visit: https://wordpress.com/me/purchases/add-payment-method
3: You'll only see one payment method here.

### What you expected to happen

All of WordPress.com's available payment methods should be on this screen, based on the same information we use to determine things such as currency.

### What actually happened

See above - there is only an option to add a credit card.

### Context

This turned out not to be the issue in 24396171-hc, however it was in this interaction that I became aware of it. My own testing indicates the same. However, if I log into an account with active subscriptions, I can add a method such as PayPal without an issue.

### Other notes

This may create some confusion for users when they're attempting to make preparations for purchasing upgrades. For example, if I'm a user, I may go to Purchases > Payment Methods to make sure I have the method I need to renew my subscription already on my account. If I don't, my first instinct will be to add it there. Then, it's a matter of switching all of my subscriptions to it. It may not occur to me if I haven't used the platform for long that I should add it to a subscription for it to be valid for use with every other subscription I have.

If this should be a feature request and not a bug report, then feel free to either 1: let me know or 2: relabel it accordingly.

### Reproducibility

Consistent

### Available workarounds?

There is no user impact

### Workaround details

Users can add other payment methods when they go through the checkout screen. Or, if they have a credit card, they can add it, purchase what they need, then switch to another payment method.

**Comment by synora** [comment]
Hi @arinoch,

This seems more like an enhancement request rather than a bug. Our public facing support doc says this: 

> Adding a payment method to your account is only possible for credit card payments. If you wish to add a new PayPal account for future renewals, you can add PayPal from the subscription’s settings page.

Link: https://wordpress.com/support/payment/#add-a-credit-card-to-your-account

**Comment by arinoch** [comment]
Should I resubmit this accordingly, then? I can close this issue and reopen it as a feature request if that’s the proper course. Thank you for the free education.

**Comment by synora** [comment]
Nope. No need to resubmit it. I've switched the labels so everything's all good now. 😄

**Comment by arinoch** [comment]
Thanks. It wouldn’t have bothered me to resubmit, but if you’re going to save me work, I’m not going to say no. 😊
